recipes/cfgfile/0.2.8.2.1/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class CfgfileConan(ConanFile):
+    name = "cfgfile"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/igormironchik/cfgfile.git"
+    license = "MIT"
+    description = "Header-only library for reading/saving configuration files with schema defined in sources."
+    generators = "cmake"
+    topics = ("conan", "cfgfile", "configuration")
+    settings = "os", "arch", "compiler", "build_type"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "3.5",
+            "apple-clang": "10"
+        }
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=os.path.join(self._source_subfolder, "generator"), build_folder=self._build_subfolder)
+        return self._cmake
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "14")
+
+        compiler = str(self.settings.compiler)
+        if compiler not in self._compilers_minimum_version:
+            self.output.warn("Unknown compiler, assuming it supports at least C++14")
+            return
+
+        version = tools.Version(self.settings.compiler.version)
+        if version < self._compilers_minimum_version[compiler]:
+            raise ConanInvalidConfiguration("cfgfile requires a compiler that supports at least C++14")
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+        self.copy("*.hpp", src=os.path.join(self._source_subfolder, "cfgfile"), dst=os.path.join("include", "cfgfile"))
+        cmake = self._configure_cmake()
+        cmake.install()
```

Comment:
```suggestion
        cmake.install()

    def package_info(self):
        bin_path = os.path.join(self.package_folder, "bin")
        self.output.info("Appending PATH env var with : {}".format(bin_path))
        self.env_info.PATH.append(bin_path)
```

Reply:
Possibly, but not sure that this is required. I successfully use `cfgfile.generator` in `test_package`... Why do we need to expand PATH environment here?

Reply:
It's a convention we have to make it easier

Reply:
test package uses run_environment ...